ads-and-adblocking.rst

```diff
@@ -0,0 +1,165 @@
+.. post:: April 10, 2018
+   :tags: advertising, privacy, sustainability, business, adblock
+   :author: David
+   :location: SAN
+
+.. TODO: update date
+
+Ads and Ad blockers at Read the Docs
+====================================
+
+Last week, we talked about how :doc:`ethical advertising works <ethical-advertising-works>`.
+Read the Docs has a number of sources of revenue from our `paid product`_
+to donations but by far the largest source of revenue comes from advertising.
+Our premium audience of professional and amateur software developers engaged
+in reading about the software they use has proved to be quite valuable to
+advertisers. Furthermore, we have been able to build our advertising model
+while **respecting user privacy** which our audience frequently and repeatedly
+tells us is very important to them. We declared we:
+
+* Never let advertisers run scripts on our site and always
+  host ad images ourselves. [#f1]_
+* Never provide data dumps of personal data to advertisers.
+* Only target advertising based on general geography [#f2]_ and the details
+  of the page being visited not a trove of behavioral data about visitors.
+
+Read the Docs `continues to grow <read-the-docs-2017-stats>`_ by every metric
+despite not taking the all too typical route of cashing in
+without considering the consequences on users.
+With all this success, it was only a matter of time before we attracted
+the attention of ad blockers.
+
+.. _paid product: https://readthedocs.com/pricing/
+
+
+All about ad blockers
+---------------------
+
+Ad blockers fulfill a legitimate need to mitigate the
+`significant downsides of advertising`_ from tracking across the internet,
+security implications of third-party code,
+and impacting the UX and performance of sites.
+Read the Docs runs a one-site ad network but most of our developers run
+ad blockers. That should tell you what you need to know about the ad industry.
+That's why we built the ad network we wanted to exist with
+no tracking and no behavioral targeting.
+
+According to a `2017 report from PageFair`_ (pdf), a company that specializes
+in quantifying ad blocking, 11% of global web users run an ad blocker.
+Considering that Read the Docs' core audience is tech-savvy, privacy-conscious
+developers, it should be no surprise that our number is higher.
+
+Most ad blockers work by subscribing users to "filter lists" that have rules
+to block ads or other content based on CSS selectors or URL patterns.
+The largest and most popular of these filter lists is called the `EasyList`_.
+Read the Docs' success in advertising led us to get added to the EasyList.
+
+.. _significant downsides of advertising: https://docs.readthedocs.io/en/latest/ethical-advertising.html#ethical-info
+.. _2017 report from PageFair: https://pagefair.com/downloads/2017/01/PageFair-2017-Adblock-Report.pdf
+.. _EasyList: https://easylist.to/
+
+
+Ad blocker fallout
+------------------
+
+.. figure:: img/2018-readthedocs-adblocker-fallout.png
```

Comment:
Would be good to include the numbers in this graph, unless there's a reason not to?